README.md

```diff
@@ -1,320 +1,64 @@
-```sh
-nano ~/.celestia-full/config.toml
-```
-```sh
-...
-[P2P]
-  ...
-  MutualPeers = ["/ip4/<ip>/tcp/2121/p2p/<pubKey>"] #add multiaddresses of other celestia full nodes
-  PeerExchange = true #change this line to true. Be default it's false
-  ...
-...
-```
+<i>Conceptually</i>, one can't live without another as we need every new block to be erasure coded in order to do data availability sampling from `Celestia Light Clients`
 
-3. Start the full node
-```sh
-celestia full start
-```
-Now, the celestia full node will start syncing headers and storing blocks from celestia app. 
+Technically, Celestia Application and Celestia Full Node are operating on different processes of <b>one instance</b>(e.g. server). Let's break down what each of the parties do. 
 
-<u>Note: At startup, we can see the `multiaddress` from celestia full node. This is <b>needed for future light client</b> connections and communication between celestia full nodes</u>
+Celestia Application(CA) takes care of these points: 
+- Consensus
+- State Interaction
+- Block Production
+- Communication between Celestia Application instances only!
 
-Example:
-```sh
-/ip4/46.101.245.50/tcp/2122/p2p/12D3KooWKNBZvF93L92aTYs6jRyozicGmuu3cF9gotMtUCeHAPYN
-```
+Celestia Full Node(CFN) takes care of these points: 
+- Sync blocks from Celestia App
+- Erasure Codes the blocks
+- Serve headers/shares to Celestia Light Clients
+- Communication between Celestia Full Nodes and Celestia Light Clients
 
-### Light Client Configuration
-To start the light client, we need to know 2 variables:
-- Trusted peer’s multi address to connect to (a celestia full node is the case here)
-- Trusted block hash from celestia-app
+Communication of CA and CFN processes happens over the RPC.
 
-1. Initialize the light client
+### Celestia Light Client
+Celestia Light Client(CLC) lives only in [celestia-node](https://github.com/celestiaorg/celestia-node) repo. What does CLC do: 
+1. Connects to a `Celestia Full Node`.
+2. Syncs the headers.
+3. Does data availability sampling(DAS) afterwards.
 
-```sh
-celestia full init --headers.trusted-peer <full_node_multiaddress> --headers.trusted-hash <hash_from_celestia_app>
-```
+> Note: `Celestia Light Clients` are not communicating between each other and only communicates between `Celestia Full Nodes`
 
-Example: 
 
-```sh 
-celestia light init --headers.trusted-peer /ip4/46.101.245.50/tcp/2122/p2p12D3KooWKNBZvF93L92aTYs6jRyozicGmuu3cF9gotMtUCeHAPYN --headers.trusted-hash F10679041E3D55363405C1D3080B91004BCAE471F35F1FBABC345B8237AEFDA2 
-```
 
-2. Start the light client
-```sh
-celestia light start
-```
-Now, the celestia light client will start syncing headers and do data availability sampling(DAS) from the full node.
+## Installation guide
+> Caveat: Make sure that you have at least 50+ Gb of free space to safely move on with this guide
```

Comment:
We need an explicit explanation that the full node and light client should be executed on different machines